components/00-settings/_vars.scss

```diff
@@ -64,6 +64,7 @@ $z-layers: (
   'default': 1,
   'skiplink': 5,
   'mijn-gent': 99,
```

Comment:
```suggestion
  'mijn-gent': 6,
```

Reply:
Are you sure about this @hstandaert ?

Reply:
I might have to update them again when making the mobile menu anyway.